sig/handwritten/app/helpers/sessions_helper.rbs

```diff
@@ -0,0 +1,2 @@
+module SessionsHelper : ActionController::Base
```

Comment:
This also describes the module-self-type of the `SessionsHelper` module. Actually, this module is included by `ApplicationController`. But Steep raises an error by circular definition if we set it. Therefore I used the `ActionController::Base` to avoid the error. This is a minor tip.